features/admin.server-configurations.v1/pages/account-disable-page.tsx

```diff
@@ -0,0 +1,235 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Grid from "@oxygen-ui/react/Grid";
+import { AppConstants } from "@wso2is/admin.core.v1/constants/app-constants";
+import { history } from "@wso2is/admin.core.v1/helpers/history";
+import { AlertLevels, IdentifiableComponentInterface } from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import { EmphasizedSegment, Hint, PageLayout } from "@wso2is/react-components";
+import { AxiosError } from "axios";
+import React, { FC, FormEvent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch } from "react-redux";
+import { Dispatch } from "redux";
+import { Checkbox, CheckboxProps } from "semantic-ui-react";
+import { getConnectorDetails, updateGovernanceConnector, useGetGovernanceConnectorById } from "../api/governance-connectors";
+import { ServerConfigurationsConstants } from "../constants/server-configurations-constants";
+import {
+    ConnectorPropertyInterface,
+    GovernanceConnectorInterface,
+    UpdateGovernanceConnectorConfigInterface,
+    UpdateGovernanceConnectorConfigPropertyInterface
+} from "../models/governance-connectors";
+
+/**
+ * Props for the Account Disable configuration page.
+ */
+type AccountDisablePageInterface = IdentifiableComponentInterface;
+
+/**
+ * Account Disable configuration page.
+ *
+ * @param props - Props injected to the component.
+ * @returns Account Disable configuration page.
+ */
+export const AccountDisablePage: FC<AccountDisablePageInterface> = (
+    props: AccountDisablePageInterface
+): ReactElement => {
+
+    const { [ "data-componentid" ]: componentId = "account-disable" } = props;
```

Comment:
Lets move the destructuring into the function params.

```tsx
export const AccountDisablePage: FC<AccountDisablePageInterface> = (
    { [ "data-componentid" ]: componentId = "account-disable" }: AccountDisablePageInterface
): ReactElement => {
```

Reply:
Fixed with c0195b74e9b173f04437faa6b151fb33c0d639f1